### ISSUE TITLE
Apple M1 support 

### ISSUE THREAD

**Issue by ktimothy** [browserless/chrome#1629]
**Is your feature request related to a problem? Please describe.**
I need to run browserless/chrome locally, but unable to do that on my Mac with Apple M1 chip, because the image is built for `linux/amd64` platform:

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

**Describe the solution you'd like**
I would like to have a docker image built for `linux/arm64/v8` platform.

**Describe alternatives you've considered**
-

**Additional context**
-

**Comment by joelgriffith** [comment]
Yeah, this is unfortunately something that might take a bit to get to due to the many dependencies involved. Always happy to review a PR
